src/Domain/Tags.hpp

```diff
@@ -227,13 +227,21 @@ struct DetInvJacobianCompute : db::ComputeTag,
 /// Base tag for boundary data needed for updating the variables.
 struct VariablesBoundaryData : db::BaseTag {};
 
+// @{
 /// \ingroup DataBoxTagsGroup
 /// \ingroup ComputationalDomainGroup
 /// The set of directions to neighboring Elements
 template <size_t VolumeDim>
-struct InternalDirections : db::ComputeTag {
+struct InternalDirections : db::SimpleTag {
   static constexpr size_t volume_dim = VolumeDim;
-  static std::string name() noexcept { return "InternalDirections"; }
+  using type = std::unordered_set<::Direction<VolumeDim>>;
+};
+
+template <size_t VolumeDim>
+struct InternalDirectionsCompute : InternalDirections<VolumeDim>,
+                                   db::ComputeTag {
+  static constexpr size_t volume_dim = VolumeDim;
+  using base = InternalDirections<VolumeDim>;
   using return_type = std::unordered_set<::Direction<VolumeDim>>;
```

Comment:
I haven't been following the DataBox-related changes all that well, but do you still need `return_type` as it is presumably always the same as `type`?

Reply:
At the moment it is still needed, but it will be disappearing soon...